main.go

```diff
@@ -14,19 +14,21 @@ import (
 	"github.com/prometheus/prometheus/storage/metric"
 	"github.com/prometheus/tsdb"
 	"github.com/prometheus/tsdb/labels"
+	"gopkg.in/cheggaaa/pb.v1"
 )
 
 func main() {
 	v1Dir := flag.String("v1-dir", "./data-v1", "Path to the v1 storage directory.")
 	v2Dir := flag.String("v2-dir", "./data-v2", "Path to the v2 storage directory.")
 	lookback := flag.Duration("lookback", 15*24*time.Hour, "How far back to start when exporting old data.")
 	step := flag.Duration("step", 15*time.Minute, "How much data to load at once.")
+	v1HeapSize := flag.Uint64("storage.local.target-heap-size", 2000000000, "How much memory to use for v1 storage in bytes")
```

Comment:
In this context, the old flag name is a bit misplaced... how about `-v1-target-heap-size`?

If we get several more flags, we can think about more namespacing again...

Reply:
Fine with me, I was mostly thinking copy-n-paste from Prometheus server flags.